x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/service/ServiceAccountToken.java

```diff
@@ -130,6 +130,11 @@ public void close() {
         secret.close();
     }
 
+    @Override
+    public String toString() {
+        return getQualifiedName();
+    }
```

Comment:
I am pretty sure we had this method previously but somehow it seemed to be lost in between rebases refactors, rebases and merges.